lib/sensors/stripe_sensor.cpp

```diff
@@ -0,0 +1,26 @@
+#include "stripe_sensor.hpp"
+
+namespace hyped::sensors {
+
+Keyence::Keyence(core::ILogger &log, const std::uint8_t newPin) : pin(newPin), logger_(log)

```

Comment:
`new_pin`. Function arguments should be in snake case.

Reply:
also `&logger` and `logger_(logger)`